Dockerfile

```diff
@@ -1 +1 @@
-FROM pyiron/pyiron-image:latest
+FROM pyiron/pyiron:2020-10-15
```

Comment:
The problem is that mybinder does not accept latest tag. There has to be a tag other than latest.

Reply:
Ok, then we keep the tag for now - maybe it is reasonable to create a separate repository for mybinder, it could be the same as the one for the example notebooks. But that is a different topic, for now we just keep the static link.